db/migrate/20210506231047_create_stocks.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateStocks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :stocks do |t|
+      t.string :ticker
+      t.string :name
```

Comment:
It would be useful to be able to rely on `not null` constraints 

```suggestion
      t.string :name, null: false
```